sample-lnd.conf

```diff
@@ -1295,6 +1296,13 @@ litecoin.node=ltcd
 ; Specify the timeout to be used when opening the database.
 ; db.bolt.dbtimeout=60s
 
+[sqlite]
```

Comment:
Have we taken care of a scenario where a user with an existing node adds `sqlite` option via the config?
Ideally, in this scenario the node should throw an error at the startup informing user that "kvdb cannot be migrated to sqlite yet, you'll not be able to add a sqlite option for an existing node"

Once migration option is in place, this error can be removed and migration can be triggered instead with this option for an existing node, or a specific flag provided to force a db migration

Reply:
it's a bit tricky to do that I think cause what if the user was using etcd or postgres before? Then how would we know that they were previously had a different backend?

Or do you just mean in the case where they were using bbolt before? Tricky part about that is that they may have changed their `datadir` between restarts. So the best we could do is do a best effort check of "is there a channel.db in their current data dir already" type of thing I think

Reply:
I am trying to avoid a scenario, where an unsuspecting user of an existing node adds sqlite and ends up with a new node database. If that's a possibility with this config option, we should mitigate for it

Reply:
resolving this comment thread as the issue is addressed in the last commit